packages/mono.rb

```diff
@@ -3,40 +3,32 @@
 class Mono < Package
   description 'Mono is a software platform designed to allow developers to easily create cross platform applications part of the .NET Foundation.'
   homepage 'http://www.mono-project.com/'
-  version '6.8.0.123-1'
+  version '6.12.0.107'
   compatibility 'all'
-  source_url 'https://download.mono-project.com/sources/mono/mono-6.8.0.123.tar.xz'
-  source_sha256 'e2e42d36e19f083fc0d82f6c02f7db80611d69767112af353df2f279744a2ac5'
+  source_url 'https://download.mono-project.com/sources/mono/mono-6.12.0.107.tar.xz'
+  source_sha256 '61f3cd629f8e99371c6b47c1f8d96b8ac46d9e851b5531eef20cdf9ab60d2a5f'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/mono-6.8.0.123-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/mono-6.8.0.123-1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/mono-6.8.0.123-1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/mono-6.8.0.123-1-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '8e62055873099e3f943dd33789274c3db2d5436ddec704c72c8cf758d24f6ee5',
-     armv7l: '8e62055873099e3f943dd33789274c3db2d5436ddec704c72c8cf758d24f6ee5',
-       i686: 'd0442cd9a809387cf79cb6a605018244704ccd25333fcc33f1e8440fc0c52a04',
-     x86_64: '46077f0cefcfc389006e730755ec9832a7624d67b9285b27410655cd65b55083',
-  })
+  depends_on 'bc'
+  depends_on 'libgdiplus'
+  depends_on 'python3' => :build
```

Comment:
This should be a core package.

Reply:
It isn't though, weirdly enough. Or at least it doesn't get installed by the install script anymore.